packages/expo-store-review/android/src/main/java/expo/modules/storereview/StoreReviewModule.kt

```diff
@@ -49,7 +49,7 @@ class StoreReviewModule(private val mContext: Context)
           if (task.isSuccessful) {
             promise.resolve(null)
           } else {
-            promise.reject(null)
+            promise.reject("E_STORE_REVIEW_FAILED", "Android ReviewManager task failed")
```

Comment:
```suggestion
            promise.reject("ERR_STORE_REVIEW_FAILED", "Android ReviewManager task failed")
```
From what I can tell looking at recent changes in `expo/expo` this is currently the recommended way of tagging errors, see
https://github.com/expo/expo/blob/bce9476185a05a9077e76eb58ab6f7b2f41d5d64/CHANGELOG.md#L54-L56